internal/db/db.go

```diff
@@ -104,6 +109,29 @@ func (db *dB) GetCompose(jobId string, orgId string) (*ComposeEntry, error) {
 	return &compose, nil
 }
 
+func (db *dB) GetComposeImageType(jobId, orgId string) (string, error) {
+	ctx := context.Background()
+	conn, err := db.Pool.Acquire(ctx)
+	if err != nil {
+		return "", err
+	}
+	defer conn.Release()
+
+	result := conn.QueryRow(ctx, sqlGetComposeImageType, orgId, jobId)
+
+	var imageType string
+	err = result.Scan(&imageType)
+	if err != nil {
+		if errors.As(err, &pgx.ErrNoRows) {
```

Comment:
Should this be `errors.Is` like it is here:
https://github.com/osbuild/osbuild-composer/commit/9def5455708f7e247fe4103d47e00a7beb52169b